study_hour/react-client/src/actions/location.js

```diff
@@ -0,0 +1,22 @@
+import axios from "axios";
+import history from '../history';
+
+export const LOCATION_FOUND = 'location_found';
+export const LOCATION_ERROR = 'location_error';
+
+export function foundLocation(location){
+  return (dispatch) => {
+    localStorage.setItem('user', response.data.token);
```

Comment:
Why?

Reply:
Was planning on using redux to save user's location. Not sure if it's still needed.